src/fidesops/graph/config.py

```diff
@@ -187,6 +188,19 @@ def parse(path_str: str) -> FieldPath:
         """Create a FieldPath from a dot-separated input string"""
         return FieldPath(*path_str.split("."))
 
+    def retrieve_from(self, input_data: Dict[str, Any]) -> Any:
+        """Retrieve input data along the FieldPath
+
+        Example:
+            input_data = {"A": {"B": {"C": 2, "D": 3}}}
+            field_path = FieldPath("A", "B", "C")
+            field_path.retrieve_from(input_data) = 2
+        """
+        sub_data = copy.deepcopy(input_data)
+        for level in self.levels:
+            sub_data = sub_data[level]
+        return sub_data
+
```

Comment:
This is vulnerable to missing levels - in the documented case `retrieve_from({"a":{"b":{}} ` will raise an error. There's also a pydash method that handles this, see [here](https://pydash.readthedocs.io/en/latest/api.html?highlight=get_#pydash.objects.get)

Reply:
Good point @stevenbenjamin, there's a similar vulnerability in `filter_data_categories`, this pydash method looks great.